style.css

```diff
@@ -0,0 +1,300 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto');
+
+html {
+	box-sizing: border-box;
+}
+
+body * {
+	font-family: Roboto, Arial, sans-serif;
+}
+
+*, *:before, *:after {
+    box-sizing: inherit;
+}
+
+header {
+	height: 70px;
+	width: 100vw;
+	top: 0;
+	background-color: white;
+	position: fixed;
+	left: 0;
+}
+
+a {
+	text-decoration: none;
+}
+
+#navbar {
+	display: flex;	
+	justify-content: space-between;
+	margin-top: 1em;
+	margin-right: 2em;
+	margin-left: 2em;
+}
+
+#brand-name {
+	font-size: 1.2em;
+}
+
+.fa-youtube {
+	color: red;
+	padding: 0.1em;
+}
+
+#icons-right i {
+	padding: 0 0.5em;
+}
+
+.avatar {
+	font-size: 2em;
+}
+
+#avatar {
+	font-size: 2em;
+}
+
+.fa-bars {
+	margin-right: 1em;
+}
+
+i {
+	color: #A0A0A0;
+	font-size: 1.2em;
+}
+
+
+#search {
+	width: 30rem;
+}
+
+#main {
+	display: flex;
+	padding: 50px 0;
+	margin-top: 3em;
+	padding-left: 3em;
```

Comment:
Could replace to the padding shorthand `padding: 50px 0 50px 3em`

Reply:
Incorporated recommendations in pull request #8 and merged into master. Proceeding with project form submission. Since this new-master branch was created solely for a code review request, it will be deleted.